### DISCUSSION TITLE
Repository naming

### DISCUSSION THREAD

**Discussion by badcel** [gircore/gir.core#40] [General]
The name gir.core has very little meaning to people as it is only a file format which describes bindable c libraries.

Is there a name available which is meaningful and is not limited to Gtk but the eco system of gobject libraries?

**Comment by knuxbbs** [comment]
I doesn't have any problem with the "gir" in gir.core name. But as .NET will remove the "core" from the future releases, I think that 'gir-sharp' or 'girsharp' are good alternatives, that are in line with other community projects. Maybe "gobject-sharp" is more representative.

> 
> Is there a name available which is meaningful and is not limited to Gtk but the eco system of gobject libraries?

I only think in glib.

**Reply by badcel** [reply]
Yes it is quite hard to find a good name. I agree to your points. And somehow this project is not only going to be a wrapper but some kind of foundation for other library authors.

Perhaps we should start to think about a organization name first as the project structure could change.
